README.md

```diff
@@ -17,27 +25,28 @@ The BDD syntax popularized by Cucumber is language-neutral, and easy for even no
             └──test
                 └──java
                 |   └──runner
-                |       KarateRunner.java
+                |      └──KarateRunner.java
                 └──resources
-                    └──karate
-                        └──features
-                            example.feature
-                            healthcheck.feature
-                        └──services
-                            └──dao
-                            └──data
-                            └──query
-                        └──support
-                            schemas.json
-                            env_data.json
+                    └──features_unitary
+                    |     └──functionality
+                    |       └── example.feature
+                    └──features_integration
+                    |     └──functionality
+                    |       └── example.feature
```

Comment:
shouldn't be something like:
```
features
 |_ unit
 |_ integration
```

Reply:
+1

Reply:
Change name

Reply:
I think the route names are changed now on every file but we should still update this readme